webtau-utils/src/main/java/com/twosigma/webtau/utils/StringUtils.java

```diff
@@ -80,6 +80,14 @@ public static String concatWithIndentation(String prefix, String multilineText)
                 "");
     }
 
+    public static boolean nullOrEmpty(String s) {
+        return s == null || s.trim().isEmpty();
```

Comment:
I see optimization you have done here. We do change behavior slightly. 
Maybe we should distinct empty vs blank? 
I wonder if someone can actually send "          " back and it will be swept out by our new logic.